src/main/java/org/axion/axion_api_gateway/UserDTO.java

```diff
@@ -0,0 +1,28 @@
+package org.axion.axion_api_gateway;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class UserDTO {
+
+    private String userId;
+    private String username;
+    private String role;
+
+    public UserDTO() {
+    }
+
+    @JsonProperty("userId")
```

Comment:
I think all these 3 @JsonProperty annotations are not required if the field name is the same, we can skip it.

Reply:
Jackson needs all three properties to proceed correctly.